intel_extension_for_deepspeed/xpu_accelerator.py

```diff
@@ -8,6 +8,15 @@ class XPU_Accelerator(DeepSpeedAccelerator):
     def __init__(self):
         self._name = 'xpu'
         self._communication_backend_name = 'ccl'
+        def _check_and_mapping_mpich_env():
+            import os
+            if  "RANK" not in os.environ and "PMI_RANK" in os.environ:
+                os.environ['RANK'] = os.environ.get('PMI_RANK')
```

Comment:
why we need to set the env in idex?

Reply:
Match Megatron, or we need to modify Megatron. I think it is better to modify idex.